src/lib/mocks/clerk-mock.tsx

```
@@ -0,0 +1,201 @@
+  openSignUp: () => {},
+  openUserProfile: () => {},
+  signOut: () => Promise.resolve()
+});
+
+// ============================================================================
+// MOCK COMPONENTS
+// ============================================================================
+
+export const SignInButton = ({ children }: { children: React.ReactNode }) => (
+  <button className="px-4 py-2 bg-pink-500 text-white rounded">
+    {children || 'Entrar (Dev Mode)'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)